contracts/lib/OrderCombiner.sol

```diff
@@ -727,6 +727,9 @@ contract OrderCombiner is OrderFulfiller, FulfillmentApplier {
             }
         }
 
+        //trigger the execution of the orders first so we have atomic settlement of the orders
```

Comment:
Let’s revise this comment to `// Trigger execution prior to post-execution checks.`